### ISSUE TITLE
content: metadata/*ms/measurements.html

### ISSUE THREAD

**Issue by nathcast** [Rothamsted-Ecoinformatics/eRA#3]
- [ ] The Measurement Page will be a combination of http://www.era.rothamsted.ac.uk/Met/rothmet and

http://www.era.rothamsted.ac.uk/Met/instrumentdescription 
and should sit on the Met Station Page where Design is in the Experiment Page. 

- [x] rms/measurements (MG)
- [x] wms/measurements (MG)

- [ ] wms- line 254 - CLOUD <b>[NEED TO FIND OUT WHEN THIS STOPPED Appears in WOBMET header file 1962.]</b>

- [x] wms- line 304 <p>From 1999 WINDSP has been calculated as an average over 10 minutes from 8.50 to 9am **[CHECK WITH TONY]**.

- [x] bms/measurements
- [x] sms/measurements
 

- [ ] rms add early visibility codes (pre 1960s)

- [x] Buck DOI link broken, keep link as journal are trying to fix this. Check journal website

- [ ] bms. We used Met Office data for 5 variables Aug 2009 - May 2012. We have shown this a (c) Crown Copyright Met Office in the website. How to show this, so it can be used. @richardostler . 

- [ ] Check if Relativel Humidity was measured at wms 1928-70 by checking photocopied monthly data sheets (in cupboard in printer room). 

- [x] Testing

- [ ]

**Comment by nathcast** [comment]
GIT only accepts PDFs and I Have not managed to make a pdf of the page that looks like the final product: so here a pdf of the table as it is now, 

[pdfera_RMS_measurements.pdf](https://github.com/Rothamsted-Ecoinformatics/eRA/files/4455999/pdfera_RMS_measurements.pdf)

Please edit the file measurements.html that I put in TEAMS > eRA>web site update > FILES tab> webPages>rms/measurements.html

![Annotation 2020-04-22 131257](https://user-images.githubusercontent.com/36696807/79980567-3576d900-849b-11ea-9f62-918661278044.png)


and let me know when that is ready

**Comment by mglendining** [comment]
I am starting to look at this properly now. It is quite a bit of work, as there is a lot of duplication and missing data. But worth doing, as it is very messy.

**Comment by mglendining** [comment]
I'll start with Rothamsted, which is the biggest, and has most information. Woburn should be fairly straightforward after that, but we may need to get hold of information from others for Brooms Barn.

**Comment by Perrymas** [comment]
Let me know in what way i can help, so not to overlap with you.

Sarah

________________________________

**Comment by mglendining** [comment]
Hi I have finished my updates to the Rothamsted met data measurements. @nathcast do you want to see if they are OK. There are a few queries for you, eg the link to Buck doesn't work, and can we have a link to the conversion factos? Then perhaps Sarah can have a look, to see if it makes sense. I'll do Woburn, as I have all the data to hand, from doing Rothamsted.

**Comment by Perrymas** [comment]
OK, well done, thanks margaret, i will take a look,
cheers

sarah

________________________________

**Comment by Perrymas** [comment]
This was a big effort to collate this together I can see, Margaret, many thanks.

I have made some edits to the Roth met data measurements and saved as -sp.txt suffix. I have suggested different titles, such as 'Methods of measurement' instead of 'How is it recorded?' - is that ok? I have edited some grammar, e.g. instead of 'so that they can easily be removed' to 'enabling easy removal'.  I have changed 'in vacuo' to 'in a vacuum' - is that ok? I have edited some minor bits with missing commas and missing 'bolds' throughout. 

The different RAIN measurements are so confusing - can be cull some of them Margaret?

 @mglendining  @nathcast

**Comment by mglendining** [comment]
Hi Sarah
I have incorporated all your changes I think, thanks for that.
Good idea to try and rationalize the rainfall, but maybe we need some more time to do that? Replace RAIN and RAIN5 with another composite dataset.

I have now finished the WOBURN equivalent, if you could have a look at that please.
Its in wms

Thanks a lot

Margaret

________________________________

**Comment by Perrymas** [comment]
Thanks margaret, i will check woburn.

Yes, a composite dataset is a good idea. We can always keep all that inf o as a foot note somewhere.

sarah

________________________________

**Comment by mglendining** [comment]
@nathcast in the Rothmet data measurments, I would lilke to completely delete the SUNMAX section, and hide it from being displayed in Rothmet. It was only measured 1915-35, and I don't have any methodology information. 
There are a few links required, and the Buck 1981 refernce no longer works. There are still a few bits of information that we are waiting on, eg visibility codes, but I can't do anymore until I go back into Rothamsted now. Could you have a look at the revised version, and make the links? Then I think it is done. Thanks Margaret

**Comment by nathcast** [comment]
> I would lilke to completely delete the SUNMAX section, and hide it from being displayed in Rothmet. It was only measured 1915-35, and I don't have any methodology information.

I have commented our SUNMAX in RMS/measurements

**Comment by nathcast** [comment]
> There are a few links required,
rms/measurements

- [x]  (see <a href="info.php?FileName=CF.html">Conversion factors</a>)

- [x] (see <a href="info.php?FileName=derived_variables.html"> derived variables<a>)

- [x] Make the problem with BUCKS refenrece a separate issue at

**Comment by nathcast** [comment]
Regarding wms: 

- [x] I have commented out RDUR

- [x] added the TODs tothe top of this issue as check boxes

- [x] conversion factor links
- [x] derived variable links

**Comment by mglendining** [comment]
Thank you. Please can we also add 'find Visibility Codes for 1928-1961: A-E, G and X.' for Wobmet.

**Comment by mglendining** [comment]
I have commented out the comment on earlier visibility codes. for wms

**Comment by mglendining** [comment]
I have made a start on the Brooms Barn measurements. Question: much of the data between 2009 and 2012 was supplied by the met office. We should put this in somewhere. I have started by putting (C) Met Office 2009-2012. Then there are other odd days and weeks. Should this also be included? This is probably a question for Richard too. It also applies to Woburn, though not as much.

**Comment by mglendining** [comment]
I have finished the first version of the Brooms Barn met data measurements. I need to check the measurment of WETB and the diameter of the rain gauge, otherwise all done. @Perrymas  can you take a look? It is called bms.

**Comment by mglendining** [comment]
I have made some small updates to the rms and wms measurement pages.

**Comment by mglendining** [comment]
I have finished the bms measurement pages, apart from the issue of how to deal with the Met Office data. Currently shown as crown copyright 2009-2012. @nathcast I would just like @Perrymas and Tony Scott to look at it.

**Comment by mglendining** [comment]
Data quality of met measurments. The equipment is regularly calibrated by Campbell Scientific, and there are probably other routine checks carried out. Should we have a separate section on Data Quality with the met data? @nathcast @richardostler @Perrymas

**Comment by nathcast** [comment]
> Data quality of met measurments. The equipment is regularly calibrated by Campbell Scientific, and there are probably other routine checks carried out. Should we have a separate section on Data Quality with the met data? @nathcast @richardostler @Perrymas

PLease go ahead and add that in the DataQualilty page which is in the default fodler

**Comment by mglendining** [comment]
@nathcast there is a stray 'S' at the start of wms. Can you get rid of it? I can't see what is causing it. Thanks M

**Comment by nathcast** [comment]
Moving content of this issue in the relevant stations issues. 
So we can close that one as tick boes will be redundant
